### PR TITLE
CSL-4: Feature - PCA About the applicants

### PR DESCRIPTION
**Pull request by RobertMcCann**
## What? 
A feature branch containing tickets relating to the 'about the applicants' section for Precursor Chemicals 

Covers tickets:
[CSL-13](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-13)       1. PCA - Licence holder details ('about the applicants' section) PR #7 
[CSL-20](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-20)	1.2 PCA - Is the premises address the same as the licence holder address? (about the applicants) PR #3
[CSL-21](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-21)	1.3 PCA - Responsible officer details (About the applicants) PR #6 
[CSL-18](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-18)	1.4 PCA - Guarantor details (about the applicants) PR #13 
[CSL-28](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-28)	1.5 PCA - Has anyone employed by your company ever received criminal convictions? (about the applicants) #14 

## Check list

- [ ] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

**Review by dk4g — APPROVED** [review]
no direct commits have been made to this PR, only through the reviewed and merged PR

File: apps/precursor-chemicals/sections/summary-data-sections.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        }
+      },
+      {
+        step: '/responsible-officer-details',
+        field: 'responsible-officer-details',
+        parse: (val, req) => {
+          const formattedResponsibleOfficerDetails = Array(
+            req.sessionModel.get('responsible-officer-fullname'),
+            req.sessionModel.get('responsible-officer-email')
+          ).join('\n');
```
Comment (jamiecarterHO) [inline apps/precursor-chemicals/sections/summary-data-sections.js:77]: We have two formats in this file for uniting several fields in an array and joining them for display in the summary. No problem as both will work, but we may want to decide on one and refactor for general readability/continuity.

**Review by jamiecarterHO — APPROVED** [review]
Approved, but with comment about dual formatting of multi-line elements in summary-data-sections

File: apps/precursor-chemicals/sections/summary-data-sections.js (hunk 1 above)
Comment (dk4g) [reply apps/precursor-chemicals/sections/summary-data-sections.js:77]: agree, I would personally prefer this option for readability:

```
const myArray = [
  req.sessionModel.get('value1'),
  req.sessionModel.get('value2'),
];

return myArray.join('\n');
```

Comment (dk4g) [reply apps/precursor-chemicals/sections/summary-data-sections.js]: Resolved by PR #21

**Pull request merged**